scripts/instance_clear.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+# Auth to kubectl
+
+https://github.com/medic/medic-infrastructure/tree/master/scripts/kubectl-authentication#mfa-authentication
+
+# Get Pod
+# TODO: how can we be sure to get the right pod?
+
+POD_ID=$(kubectl --kubeconfig=/app/cluster01-kubeconfig.yml -n gamma-dev get pods)
```

Comment:
@Hareet  My main question would be the best way to get the correct POD. There seems to be a random character set attached `dev-gamma-7677fcf668-82jvm`. Since there are currently 3 potential options would I just need to match based on the prefix?

 EX: `if(POD_ID.begins_with('dev-gamma)` then this is the id?

Reply:
I worked this out by using the kubectl selector arg